datasetComparison/src/test/scala/za/co/absa/hermes/datasetComparison/SchemaUtilsSuite.scala

```diff
@@ -66,4 +66,20 @@ class SchemaUtilsSuite extends FunSuite with SparkTestBase  {
       alignSchema(dfA, getDataFrameSelector(dfB.schema))
     }
   }
+
+  test("Test diff schema - different") {
+    val dfA = getDataFrameFromJson(spark, Seq(schemaA)).schema
+    val dfB = getDataFrameFromJson(spark, Seq(schemaB)).schema
+
+    assert(diffSchema(dfA, dfB) == List("key cannot be found in second schema"))
+    assert(diffSchema(dfB, dfA) == List("legs[StructType].conditions[StructType].price cannot be found in second schema"))
```

Comment:
What does `[StructType]` mean in the error message? Is `legs.conditions.price` descriptive enough?

Reply:
Legs is an array of struct types. 
`legs.conditions.price` might be enough. I just wanted to be descriptive enough.